packages/cli/src/commands/versions/migrate.ts

```diff
@@ -15,45 +15,54 @@
  */
 import { BackstagePackageJson, PackageGraph } from '@backstage/cli-node';
 import chalk from 'chalk';
-import { resolve as resolvePath } from 'path';
+import { resolve as resolvePath, join } from 'path';
```

Comment:
```suggestion
import { resolve as resolvePath, joinPath } from 'path';
```

just a common pattern